src/jobrunner.rs

```diff
@@ -479,34 +479,37 @@ impl JobRunner {
     }
 
     /// If the user has specified an email address, send the contents of
-    fn sendemail(&self, job: &Job, mut file: &File) {
-        if let Some(ref toaddr) = job.rconf.email {
-            let mut buf = Vec::new();
-            buf.extend(format!("Subject: snare error: {}\n\n", job.repo_id).as_bytes());
-            file.seek(SeekFrom::Start(0)).ok();
-            file.read_to_end(&mut buf).ok();
-
-            let fromea = match EmailAddress::new(format!("{}@{}", username(), hostname())) {
-                Ok(ea) => Some(ea),
-                Err(_) => None,
-            };
-
-            match EmailAddress::new(toaddr.to_string()) {
-                Ok(toea) => {
-                    let env = match Envelope::new(fromea, vec![toea]) {
-                        Ok(env) => env,
-                        Err(e) => return self.snare.error_err("Couldn't send email: {:?}", e),
-                    };
-                    let email = SendableEmail::new(env, "na".to_string(), buf);
-
-                    let mut sender = SendmailTransport::new();
-                    if let Err(e) = sender.send(email) {
-                        self.snare.error_err("Couldn't send email: {:?}", e);
-                    }
+    fn run_errorcmd(&self, job: &Job) {
+        if let Some(raw_errorcmd) = &job.rconf.errorcmd {
+            let errorcmd = errorcmd_replace(
+                raw_errorcmd,
+                &job.event_type,
+                &job.owner,
+                &job.repo,
+                &job.json_path.as_os_str().to_str().unwrap(),
+                &job.stderrout.path().as_os_str().to_str().unwrap(),
+            );
+            let mut child = match Command::new(&self.shell)
+                .arg("-c")
+                .arg(&errorcmd)
+                .current_dir(job.tempdir.path())
+                .stderr(process::Stdio::null())
+                .stdout(process::Stdio::null())
+                .stdin(process::Stdio::null())
```

Comment:
Did we want to do anything with the output of a failing errorcmd? Not worth the effort?

Reply:
How many levels deep do you want to go? :) If the `errorcmd` fails then we report that failure to the log, but I don't think it would be appropriate to send an unbounded amount of stderr/stdout there.

Reply:
I buy your reasoning. OK.